source/_posts/2020-01-29-shipit-xvii.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+title: "ShipIt Day XVII"
+author: Caitie Oder
+date: 2020-01-29
+tags: [engineering, shipit]
+---
+
+ShipIt Day, Cerner Engineering’s 24-hour hackathon, provides associates an environment to be creative, work on new projects, and network with other associates. This event is one of the many things that sit at the core of our [Engineering Culture](https://engineering.cerner.com/blog/devculture-team) at Cerner, and is rapidly growing- now being hosted globally in Romania, Sweden, India, and the UK. In October, our team had the chance to participate in the seventeenth ShipIt Day at Kansas City’s [Innovations Campus](https://engineering.cerner.com/blog/one-year-calling-innovations-home/) and managed to take home the Golden Keyboard trophy as the first place team!
+
+<div align="center">
+  <table>
+    <tr>
+      <td align="center">
+        {% img center /assets/2020-01-29-shipit-xvii/z3ro_d4y-min.png 600px %}
```

Comment:
for previous posts, the name of the file has always been the blog title and the number (e.g. shipit-xvii-1.png). doesn't really affect anything, but might be something worth discussing going forward

Reply:
@amcginn I can update if feel strongly. My 2 cents is that its smurf naming to name the files the same as the blog post since its already in a folder named after the post title. To me, a meaningful name is more useful and easier to ensure you put the right image in the right location vs having a bunch of meaningless file names

![image](https://user-images.githubusercontent.com/8029578/73197111-6642ae80-40f6-11ea-8c2d-28c2c5b8db38.png)

Reply:
I agree. just wanted to note the difference for this post